src/ui/public/directives/bread_crumbs.js

```diff
@@ -12,10 +12,11 @@ module.directive('breadCrumbs', function () {
     },
     template: breadCrumbsTemplate,
     controller: function ($scope) {
-      $scope.crumbs = chrome.getBreadcrumbs();
+      // Capitalize the first letter of each bread crumb.
+      $scope.breadcrumbs = chrome.getBreadcrumbs().map(breadcrumb => _.startCase(breadcrumb));
```

Comment:
This was originally done via CSS, but I think it's a little more discoverable if we do it in JS.